src/components/blockquote/Blockquote.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Content from '../content';
+
+const Blockquote = ({ content, entryMap, level }) => (
+    <blockquote data-type="blockquote">
+        <Content data={ { content } } entryMap={ entryMap } level={ level + 1 } />
```

Comment:
Just so I understand, any reason why we wouldn't send level+1 from the parent component, instead of adding it here? Maybe this component could just spread the `level` prop to content, and it would be responsibility of the parent to send the desired value. Thoughts?

Reply:
Because blockquote is that `level`. But its children will be "bellow" him, so the level is increased.

There might be cases where we might want to use the level to check something and since `Content` is only an orchestrator that doesn't actually render anything (just delegates) it shouldn't increase the level by itself.